website/docs/patterns/filter-patterns/partials/guidelines/guidelines.md

```diff
@@ -171,30 +171,197 @@ If applying filters to a data set yields no results, offer users an option to cl
+
+- Accommodates numerous applied filters
+- Simplifies the UI, reducing cognitive load
+- Supports complex, interactive content within the Accordion (e.g., a "Clear all filters" function)
+
+![Intermediate example of applied filters within an open Accordion](/assets/patterns/filter-patterns/applied-filters-overflow-intermediate-example-open.png)
+
+<Doc::ImageCaption @text="Moving the applied filters to an Accordion." />
+
+### Filter parameters overflow
+
+In complex data sets with numerous filterable parameters, considering establishing a hierarchy of filters by:
+
+1. Prioritizing crucial filters directly in the UI and associating them directly with the Table and data set
+2. Moving less important filter parameters to a [Flyout](/components/flyout), using [Checkbox groups](/components/form/checkbox) for each parameter and its values.
+
+This approach offers a scalable solution for highly complex data sets while communicating a natural hierarchy of filter importance.
+
+![Trigger the overflow of filter parameters with a Button](/assets/patterns/filter-patterns/filter-parameters-overflow-more-filters.png)
 
-### Sidebar overflow
+<Doc::ImageCaption @text="Use a secondary Button to trigger a Flyout." />
 
-Given that the page-level sidebar occupies the height of the viewport, take filters within a sidebar should not overflow into an interstitial component like a [Flyout](/components/flyout). Instead, utilize the height of the viewport and the consider introducing a scrollable area within the sidebar if many filters are necessary.
+![Display the overflow of filter parameters within a Flyout](/assets/patterns/filter-patterns/filter-parameters-overflow-flyout.png)
+
+<Doc::ImageCaption @text="Display the overflow of filter parameters in a Flyout" />
+
+#### Accordions within a Flyout
+
+For excessive filter parameters causing a long scroll in the [Flyout](/components/flyout), consider placing each parameter in a `flush`, `small` [Accordion](/components/accordion), and optionally, add an "Expand/Collapse All" Button. This approach gives users more control over the UI and allows for visual comparison between filter parameters.
+
+![Accordions within a Flyout containing filter parameters](/assets/patterns/filter-patterns/filter-parameters-overflow-flyout-accordion.png)
+
+<Doc::ImageCaption @text="Use Accordions within a Flyout to support an excessive number of filter parameters." />
+
+### Filter hierarchy
+
+Filter parameters vary in importance, which should be reflected in their organization in the UI. Moving frequently used filters to a [Flyout](/components/flyout) can be detrimental, as accessing common filters may require multiple steps.
+
+!!! Info
+
+Impact and priority of filters should be determined by each product and application team. User needs and priorities vary across products and user journeys, affecting the prioritization of UI elements.
+!!!
```

Comment:
[suggestion non-blocking] This feels obvious because all of this is purely recommendation-based and needs to be built by the product teams. I wonder if it needs to be said at all, but I'm curious what others think.

Reply:
Yeah, there's a bit of implication here, I'll remove it.